.github/workflows/deploy-sandbox.yaml

```diff
@@ -45,18 +45,17 @@ jobs:
         working-directory: ./src
         run: docker compose run app python manage.py collectstatic --no-input
       - name: Deploy to cloud.gov sandbox
-        uses: 18f/cg-deploy-action@main
+        uses: cloud-gov/cg-cli-tools@main
         env:
-          DEPLOY_NOW: thanks
           ENVIRONMENT: ${{ needs.variables.outputs.environment }}
           CF_USERNAME: CF_${{ needs.variables.outputs.environment }}_USERNAME
```

Comment:
Are these actually used...? They look like vestiges of a time before the repository had environments.